mvn-extension/src/test/java/org/arquillian/smart/testing/mvn/ext/ConfigLookUpTest.java

```diff
@@ -0,0 +1,78 @@
+package org.arquillian.smart.testing.mvn.ext;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Properties;
+import org.apache.maven.execution.DefaultMavenExecutionRequest;
+import org.apache.maven.execution.MavenExecutionRequest;
+import org.apache.maven.execution.MavenSession;
+import org.assertj.core.api.Assertions;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
just a detail: better is not using the annotations (runners) and just use the `Mockito.mock*` methods

Reply:
why no annotations? what is the problem?

Reply:
I didn't say that there is any problem. 
There are just two benefits when the annotation is not used:
 * it is more portable (e.g. migration to JUnit5)
 * when it is needed, we can easily change/extend the test using an additional runner (e.g. ParametrizedRunner)

This is just my approach/opinion ;-)